### PR TITLE
Moved the API key to an environment variable

### PR DESCRIPTION
**Pull request by raysarebest**
It's clearly best practice to not check an API key into version control, and thus keep it in an environment variable, and this commit takes care of that. The API key to the Digital Bible Platform should now be set in the environment as `BIBLETAGAPI_DBP_API_KEY`, and can be referenced from a `Configuration` struct as `DBPAPIKey`. If it's not set in the environment, it will default to `nil` on the struct

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -17,6 +18,7 @@ func ImportConfig() Configuration {
 		Dbname:        "bibletagapi",
 		Dbaddress:     "localhost:28015",
 		TagPostTable:  "tags",
+		DBPAPIKey:		 nil
```
Comment (dwhitena) [inline config.go]: This needs to be changed to `""` instead of `nil` because Go will barf on you if the types don't match.

**Comment by raysarebest** [comment]
Aww, we're not just passing around pointers and `nil` isn't just a `NULL` pointer? I guess Objective-C has screwed up the way I think about things. Gimme a minute, and I'll fix it

**Comment by raysarebest** [comment]
Should be good now

File: config.go [hunk 2]
```diff
@@ -17,6 +18,7 @@ func ImportConfig() Configuration {
 		Dbname:        "bibletagapi",
 		Dbaddress:     "localhost:28015",
 		TagPostTable:  "tags",
+		DBPAPIKey:		 ""
```
Comment (dwhitena) [inline config.go]: Ah one more thing here.  There should be a comma after `DBPAPIKey:       ""`.

**Comment by raysarebest** [comment]
Wow, that seems really counterintuitive, but whatever. Just a second

**Comment by raysarebest** [comment]
Should be good to go

**Pull request merged**